projects/go-lib/src/lib/components/go-table/go-table.component.ts

```diff
@@ -107,6 +107,7 @@ export class GoTableComponent implements OnInit, OnChanges, OnDestroy, AfterView
 
   private destroy$: Subject<void> = new Subject();
   private pageChange$: Subject<void> = new Subject();
+  private selectionSubscriptions: Subscription = new Subscription();
```

Comment:
I think everywhere else we've been adding a subject like `pageChange$` to track and remove subscriptions. Should we use the same here?